packages/ppf-contracts/contracts/PPF.sol

```diff
@@ -7,6 +7,23 @@ import "./open-zeppelin/ECRecovery.sol";
 contract PPF is IFeed {
     using ECRecovery for bytes32;
 
+    uint256 constant public ONE = 10 ** 18; // 10^18 is considered 1 in the price feed to allow for decimal calculations
+    bytes32 constant public PPF_v1_ID = 0x33a8ba7202230fa1cee2aac7bac322939edc7ba0a48b0989335a5f87a5770369; // keccak256("PPF-v1");
```

Comment:
Perhaps we should increment this ID to 2 now, with these changes and redeployment.

Reply:
I'd say there is no need since the contract logic hasn't fundamentally changed (and therefore signatures have the same effect) and the signature scheme is still the same.